k2/python/csrc/torch/ragged.cu

```diff
@@ -138,6 +137,24 @@ static void PybindRaggedImpl(py::module &m) {
   PybindRaggedTpl<int32_t>(m, "RaggedInt");
 }
 
+static void PybindRaggedShape(py::module &m) {
+  py::class_<RaggedShape> pyclass(m, "RaggedShape");
+  pyclass.def("tot_size", &RaggedShape::TotSize);
+  pyclass.def("num_elements", &RaggedShape::NumElements);
```

Comment:
Would you mind looking also at the Ragged binding and adding dim0(),  num_axes(), row_ids() and row_splits(), and to() ?

Reply:
also max_size(), and aything else you can see that might be useful.

Reply:
Finished previously mentioned functions. I am writing ragged_shape_test.py.